exercises/diffie-hellman/example.js

```diff
@@ -0,0 +1,47 @@
+  ];
+
+  this.getPublicKeyFromPrivateKey = function (privateKey) {
+    if (privateKey <= 1 || privateKey > this.p - 1) {
+      throw Error('Private key a must be greater than one but less than modulus parameter p!');
+    }
+    return Math.pow(this.g, privateKey) % this.p;
+  };
+
+  this.getSharedSecret = function (ourPrivateKey, theirPublicKey) {
+    return Math.pow(theirPublicKey, ourPrivateKey) % this.p;
+  };
+
+  this.validateInitialArguments = function (p, g){
+    var BIGGEST_PRIME = this.PRIMES[this.PRIMES.length - 1];
+    return p >= 2
+      && g >= 2
+      && p <= BIGGEST_PRIME
+      && g <= BIGGEST_PRIME
+      && arrIncludes(this.PRIMES, p)
+      && arrIncludes(this.PRIMES, g);
+  };
+
+  function arrIncludes(arr, a) {
+    for (var i = 0; i < arr.length; i++) {
+      if (arr[i] === a)
```

Comment:
Our preferred style for this track is to use curly braces everywhere, even for single-line `if` bodies.